### DISCUSSION TITLE
功夫熊貓跟鋼鐵人打 誰會贏 ?

### DISCUSSION THREAD

**Discussion by KozakHou** [1101-GS4719-C/Python-Community-GS4719-C#1393] [General]
我同學說是鋼鐵人，而我覺得是阿波，究竟是何者呢?

這個問題在往下延伸討論下去就是，鋼鐵人是紙老虎，因為都是特效，再者，他最後死了。

但是阿波還活著，所以結論就是，我們中國真是太厲害啦 !

**Comment by brian431** [comment]
我也這麼覺得。鋼鐵人拍攝過程都是使用電腦特效。不像功夫熊貓，是真的請到阿波本人以及蓋世五俠等等中國知名武打大師來參演的。真的打起來肯定是阿波獲勝。中國真的很厲害。

**Comment by poweichen00** [comment]
我覺得功夫熊貓會贏，因為他有無錫碎骨指，被他抓到就去了。

無錫碎骨指：https://youtu.be/NfabKk-Ki98

無錫碎骨指的PTT：https://pttcomic.com/c_chat/M.1624889562.A.1DA.html
